chart/apisix/Chart.yaml

```diff
@@ -31,9 +31,16 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.0
+version: 2.1.0
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
-appVersion: 1.16.0
+appVersion: 2.1.0
+
+dependencies:
+  - name: etcd
+    version: 5.2.1
```

Comment:
`5.2.1`? I am confused with this version value

Reply:
This is etcd Charts version ，not the version of etcd.
It is Chart dependencies here.

Reply:
In Charts, version is for Charts, appVersion is for the application.